sql/moz-fx-data-shared-prod/stripe/itemized_payout_reconciliation/view.sql

```diff
@@ -77,6 +77,16 @@ subscriptions AS (
   FROM
     `moz-fx-data-shared-prod`.subscription_platform.stripe_subscriptions
 ),
+taxes AS (
+  SELECT
+    id AS invoice_id,
+    STRING_AGG(DISTINCT currency, ", " ORDER BY currency) AS tax_currency,
+    SUM(tax_amount) AS tax_amount,
+  FROM
+    mozdata.stripe.itemized_tax_transactions
```

Comment:
Best not to reference `mozdata` views in ETL as those are deployed as a secondary step and CI/deployments can get weird failures if there are cross-dependent changes.
```suggestion
    `moz-fx-data-shared-prod`.stripe.itemized_tax_transactions
```